upgrade/patches/43_api_token.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE {{%api_token}} (
+  apt_id int unsigned NOT NULL auto_increment,
+  apt_usr_id int unsigned NOT NULL,
+  apt_created datetime NOT NULL,
+  apt_status varchar(10) NOT NULL default 'active',
+  apt_token varchar(32) NOT NULL,
+  PRIMARY KEY (apt_id),
+  KEY (apt_token)
```

Comment:
where do you say to ditch auto increment? i only said `KEY (apt_token)` should be rather `KEY (apt_token, apt_status)`

but as i see you disagree with the rest, probably can leave as is and merge when you're done.